libc/newhdrgen/yaml/math.yaml

```diff
@@ -2323,6 +2356,35 @@ functions:
     arguments:
       - type: const long double *
       - type: const long double *
+  - name: totalordermag
+    standards:
+      - stdc
+    return_type: int
+    arguments:
+      - type: double *
+      - type: double *
+  - name: totalordermagf
+    standards:
+      - stdc
+    return_type: int
+    arguments:
+      - type: float *
+      - type: float *
+  - name: totalordermagl
+    standards:
+      - stdc
+    return_type: int
+    arguments:
+      - type: long double *
+      - type: long double *
+  - name: totalordermagf128
+    standards:
+      - stdc
+    return_type: int
+    arguments:
+      - type: float128 *
+      - type: float128 *
+    guard: LIBC_TYPES_HAS_FLOAT128
```

Comment:
`totalordermag*` functions take pointers to `const` values. See section F.10.12.2 of https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3096.pdf.

Reply:
I opened #103935.